config/initializers/locations.rb

```diff
@@ -1 +1,5 @@
-Locations.geo_json_path_or_url = ENV['LOCATIONS_URL'] || 'https://locator.pensionwise.gov.uk/locations.json'
+Locations.geo_json_path_or_url = if Rails.env.test?
+                                   Rails.root.join('features', 'fixtures', 'locations.json')
```

Comment:
We should set it only once then? Removing it from `env.rb` would probably be the thing to do